server.js

```diff
@@ -98,20 +107,26 @@ function join_custom_room(ws, join_room_json) {
     // Add a prefix to the room id to indicate custom match.
     room_id = "custom_" + room_id
 
+    // Extract room settings from the passed in json.
     var deck_id = join_room_json.deck_id
+    var starting_timer = join_room_json.starting_timer
+    var enforce_timer = join_room_json.enforce_timer
     var player = active_connections.get(ws)
     player.set_deck_id(deck_id)
     var success = false
+
+    // Check whether the player is joining an existing room.
     if (game_rooms.hasOwnProperty(room_id)) {
       const room = game_rooms[room_id]
+      // Check for Player/Room version mismatch.
```

Comment:
Again, comments that just repeat what the code says exactly aren't super useful.
In this case, maybe plain "version" should be renamed to joining_player_version or something.

Reply:
renamed version to player_join_version